extensions/applicationinsights-osplugin-js/typedoc.json

```diff
@@ -0,0 +1,62 @@
+{
+    "$schema": "https://typedoc.org/schema.json",
+    "entryPoints": [ "./src/applicationinsights-osplugin-js.ts" ],
+    "exclude": [ "**/internal/**/*.ts", "node_modules/**" ],
+    "externalPattern": [
+        "**/node_modules/**",
+        "node_modules/**"
+    ],
+    "sort": [
+        "alphabetical",
+        "kind",
+        "instance-first"
+    ],
+    "basePath": "./src",
+    "sourceLinkTemplate": "https://github.com/microsoft/ApplicationInsights-JS/blob/main/{path}#L{line}",
+    "cleanOutputDir": true,
+    "excludeExternals": false,
+    "excludeInternal": true,
+    "excludePrivate": true,
+    "includeVersion": true,
+    "groupOrder": [
+        "Entrypoint",
+        "Modules",
+        "Namespaces",
+        "Enumerations",
+        "Enumeration Members",
+        "Classes",
+        "Interfaces",
+        "Type Aliases",
+        "Constructors",
+        "Properties",
+        "Variables",
+        "Functions",
+        "Accessors",
+        "Methods",
+        "References",
+        "*"
+    ],
+    "tsconfig": "./tsconfig.json",
+    "out": "../../docs/webSdk/applicationinsights-osplugin-js",
+    "readme": "none",
+    "githubPages": true,
+    "gitRevision": "main",
+    "compilerOptions": {
+        "stripInternal": true
+    },
+    "sidebarLinks": {
+        "Changelog": "https://github.com/microsoft/ApplicationInsights-JS/blob/main/RELEASES.md",
+        "Examples": "https://github.com/microsoft/ApplicationInsights-JS/blob/main/examples/README.md",
+        "Readme": "https://github.com/Microsoft/ApplicationInsights-JS"
```

Comment:
This (I think) should be the Readme of this plugin not the root one (this is one of the "fixes" we will need for the other components as well)